src/main/scala/is/hail/utils/FlipbookIterator.scala

```diff
@@ -2,6 +2,7 @@ package is.hail.utils
 
 import scala.collection.GenTraversableOnce
 import scala.collection.generic.Growable
+import scala.language.reflectiveCalls
```

Comment:
So I don't like this one.

The core issue is that we're reflectively accessing `_isValid` because it's not part of the anonymous class's type. Let's leave this out and I'll re-work this class.